Meter.h

```diff
@@ -107,7 +107,7 @@ struct Meter_ {
    char* caption;
    int mode;
    unsigned int param;
-   GraphData* drawData;
+   GraphData drawData;
```

Comment:
I don't like embedding the `GraphData` structure into the `Meter` object. If the user has never enabled the graph mode for a meter, the graph data should remain unallocated.

Reply:
The bulk data does remain unallocated though, since the `values` member is now a pointer instead of an array.  (`GraphData` itself is only a tiny handful of bytes.)